src/types.ts

```diff
@@ -12,6 +12,7 @@ export type quizTitleType = string | null;
 export type textAreaValueType = string | number | null;
 export type isFirstSessionType = boolean | null;
 type quizNavigationMode = "linear" | "non-linear";
+export type eventType = "start-quiz" | "resume-quiz" | "end-quiz"
```

Comment:
you can define it as an `enum` instead of `type`.
That way when you use it, you won't have to do this:
```js
 payload = {
    event: "start-quiz"
  }
```
But instead this
```js
 payload = {
  event: EventType.START_QUIZ
}
```

You can define an enum like this
```js
export enum backendBaseUrl {
  PROD = 'https://qmu3dnlxva.execute-api.ap-south-1.amazonaws.com/',
  STAGING = 'https://fvcgsghobb.execute-api.ap-south-1.amazonaws.com/'
}
```

and use it like this
```js
console.log(backendBaseUrl.PROD)
console.log(backendBaseUrl.STAGING)
```

This will avoid any errors that might come when writing "start-quiz" or any other event as a _string_

Reply:
yes, this is better.